lib/class-wp-rest-blocks-controller.php

```diff
@@ -33,4 +33,21 @@ public function check_read_permission( $post ) {
 
 		return parent::check_read_permission( $post );
 	}
+
+	/**
+	 * Retrieves the block's schema, conforming to JSON Schema.
+	 *
+	 * @since 4.4.0
+	 *
+	 * @return array Item schema data.
+	 */
+	public function get_item_schema() {
+		$schema = parent::get_item_schema();
+
+		// Allow all contexts to access the raw title and content of a block.
```

Comment:
Doesn't this… do exactly the opposite? We're `unset`ing the variable. Maybe there is some PHP nuance I'm missing out on 😅

Reply:
The reason that `content.raw` doesn't appear in all contexts is because, when declaring the `content.raw` property, the schema specifies `'context' => array( 'edit' )`:

https://github.com/WordPress/wordpress-develop/blob/master/src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php#L2016

`unset()`ing this `context` value means that the property will appear in all contexts.